tests/test_node_manager.py

```diff
@@ -74,14 +74,26 @@ def test_init_slots_cache(s):
     Test that slots cache can in initialized and all slots are covered
     """
     good_slots_resp = [
-        [0, 5460, [b'127.0.0.1', 7000], [b'127.0.0.1', 7003]],
-        [5461, 10922, [b'127.0.0.1', 7001], [b'127.0.0.1', 7004]],
-        [10923, 16383, [b'127.0.0.1', 7002], [b'127.0.0.1', 7005]],
+        [0, 5460, [b'127.0.0.1', 7000], [b'127.0.0.2', 7003]],
+        [5461, 10922, [b'127.0.0.1', 7001], [b'127.0.0.2', 7004]],
+        [10923, 16383, [b'127.0.0.1', 7002], [b'127.0.0.2', 7005]],
     ]
 
     s.execute_command = lambda *args: good_slots_resp
     s.connection_pool.nodes.initialize()
     assert len(s.connection_pool.nodes.slots) == NodeManager.RedisClusterHashSlots
+    for slot_info in good_slots_resp:
+        all_hosts = ['127.0.0.1', '127.0.0.2']
+        all_ports = [7000, 7001, 7002, 7003, 7004, 7005]
+        slot_start = slot_info[0]
+        slot_end = slot_info[1]
+        for i in range(slot_start, slot_end + 1):
+            assert len(s.connection_pool.nodes.slots[i]) == len(slot_info[2:])
```

Comment:
This diff should fix the problem i found here

```diff
diff --git a/tests/test_node_manager.py b/tests/test_node_manager.py
index 21add47..ed60f7a 100644
--- a/tests/test_node_manager.py
+++ b/tests/test_node_manager.py
@@ -79,20 +79,24 @@ def test_init_slots_cache(s):
         [10923, 16383, [b'127.0.0.1', 7002], [b'127.0.0.2', 7005]],
     ]
 
-    s.execute_command = lambda *args: good_slots_resp
-    s.connection_pool.nodes.initialize()
-    assert len(s.connection_pool.nodes.slots) == NodeManager.RedisClusterHashSlots
-    for slot_info in good_slots_resp:
-        all_hosts = ['127.0.0.1', '127.0.0.2']
-        all_ports = [7000, 7001, 7002, 7003, 7004, 7005]
-        slot_start = slot_info[0]
-        slot_end = slot_info[1]
-        for i in range(slot_start, slot_end + 1):
-            assert len(s.connection_pool.nodes.slots[i]) == len(slot_info[2:])
-            assert s.connection_pool.nodes.slots[i][0]['host'] in all_hosts
-            assert s.connection_pool.nodes.slots[i][1]['host'] in all_hosts
-            assert s.connection_pool.nodes.slots[i][0]['port'] in all_ports
-            assert s.connection_pool.nodes.slots[i][1]['port'] in all_ports
+    with patch.object(StrictRedis, 'execute_command') as execute_command_mock:
+        execute_command_mock.side_effect = lambda *args: good_slots_resp
+
+        s.connection_pool.nodes.initialize()
+
+        assert len(s.connection_pool.nodes.slots) == NodeManager.RedisClusterHashSlots
+
+        for slot_info in good_slots_resp:
+            all_hosts = ['127.0.0.1', '127.0.0.2']
+            all_ports = [7000, 7001, 7002, 7003, 7004, 7005]
+            slot_start = slot_info[0]
+            slot_end = slot_info[1]
+            for i in range(slot_start, slot_end + 1):
+                assert len(s.connection_pool.nodes.slots[i]) == len(slot_info[2:])
+                assert s.connection_pool.nodes.slots[i][0]['host'] in all_hosts
+                assert s.connection_pool.nodes.slots[i][1]['host'] in all_hosts
+                assert s.connection_pool.nodes.slots[i][0]['port'] in all_ports
+                assert s.connection_pool.nodes.slots[i][1]['port'] in all_ports
 
     assert len(s.connection_pool.nodes.nodes) == 6
```

The following command in `initialize` required mocking 

```
cluster_slots = r.execute_command("cluster", "slots")
```